terraform/environments/analytical-platform-ingestion/dms/data.tf

```diff
@@ -1 +1,12 @@
 #### This file can be used to store data specific to the member account ####
+
+data "aws_availability_zones" "available" {
+    filter {
+        name = "region-name"
+        values = [ "eu-west-2" ]
+    }
+}
+
+data "aws_vpc" "connected_vpc" {
+    cidr_block = local.environment_configuration.connected_vpc_cidr
+}
```

Comment:
We should look this up by name really

Reply:
Can do if you'd prefer, someone with prod access will have to tell me the name of it on prod though as I don't have permissions

Reply:
Its here https://github.com/ministryofjustice/modernisation-platform-environments/blob/main/terraform/environments/analytical-platform-ingestion/vpc.tf#L7

Reply:
Thinking about it a bit more, it might be useful to have the CIDR easily visible as it's used in the firewall rules defined in `ministryofjustice/modernisation-platform`. Are you okay if I leave the CIDR documented as a comment?